src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
```

Comment:
```suggestion
public class Truck extends Machine {
```

Reply:
Excuse me, did yopu check "solution1" branch? I dont understand where is mistakes